styles.css

```diff
@@ -0,0 +1,106 @@
+html {
+  scroll-behavior: smooth;
+}
+
+:root {
+  --theame-color: #6070ff;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 340px;
+}
+
+.logo a {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 20px;
+  color: var(--theame-color);
+  align-items: center;
+  justify-content: center;
+  margin-left: 10px;
+  text-decoration: none;
+}
+
+.logo:hover {
+  transform: scale(0.9);
```

Comment:
[OPTIONAL] There is an empty blank space to the right side of your page which is not availble on the figma design. Kindly check and fix it to make it look like the given design. :pray: 

Your design

![Screenshot from 2022-12-02 09-33-31](https://user-images.githubusercontent.com/65233860/205262267-b07954b5-013f-4e5a-a76d-fa86d1574207.png)

Figma design

![Screenshot from 2022-12-02 08-24-48](https://user-images.githubusercontent.com/65233860/205262292-c66b3547-ef44-460f-a756-c1956a707db7.png)